package.json

```diff
@@ -25,24 +25,34 @@
     "Other"
   ],
   "activationEvents": [
-    "onCommand:extension.dash.specific",
-    "onCommand:extension.dash.all"
+    "onCommand:extension.dash.selection",
+    "onCommand:extension.dash.all",
+    "onCommand:extension.dash.syntax",
+    "onCommand:extension.dash.customWithSyntax"
   ],
   "main": "./out/src/extension",
   "contributes": {
     "commands": [
       {
-        "command": "extension.dash.specific",
-        "title": "Search in Dash for current syntax"
+        "command": "extension.dash.selection",
+        "title": "Search in Dash for current selection"
       },
       {
         "command": "extension.dash.all",
         "title": "Search in Dash for all documentation"
+      },
+      {
+        "command": "extension.dash.syntax",
+        "title": "Search in Dash for current syntax"
+      },
+      {
+        "command": "extension.dash.customWithSyntax",
+        "title": "Search a custom string in Dash"
       }
     ],
     "keybindings": [
       {
-        "command": "extension.dash.specific",
+        "command": "extension.dash.selection",
```

Comment:
This is breaking change, some users may have override the shortcut with the current command name. I'd rather to keep the current command name.